src/main/java/org/fundacionjala/coding/nestor/Customer.java

```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.coding.nestor;
+
+public class Customer {
+    private String name;
+    private Rental rental;
+
+    public Customer(String name) {
+        this.name = name;
+    }
+
+    public void createRental(int daysRented) {
+        rental = new Rental(daysRented);
+    }
+
+    public void addMovie(Movie movie) {
+        rental.addMovie(movie);
+    }
+    
+    public String statement() {
+        String result = "Rental Record for " + name + "\n";
+        result += "Amount owed is " + String.valueOf(rental.getCharge()) +
+                "\n";
+        result += "You earned " + String.valueOf(rental.getFrequentRenterPoints())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Directly append the argument of String.valueOf(). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1153)